.github/workflows/ci-license.yaml

```diff
@@ -40,27 +42,7 @@ jobs:
           fetch-depth: 0
           ref: ${{ github.event.pull_request.head.sha }}
 
-      - name: Cache local Maven repository
-        uses: actions/cache@v2
-        with:
-          path: ~/.m2/repository
-          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
-          restore-keys: |
-            ${{ runner.os }}-maven-
-
-      - name: Set up JDK 1.8
-        uses: actions/setup-java@v1
-        with:
-          java-version: 1.8
-
-      # license check fails with 3.6.2 so we have to downgrade
-      - name: Set up Maven
-        uses: apache/pulsar-test-infra/setup-maven@master
-        with:
-          maven-version: 3.6.1
-
-      - name: build and check license
-        run: mvn -B -ntp -DskipTests license:check install
-          
       - name: license check
-        run: src/check-binary-license ./distribution/server/target/apache-pulsar-*-bin.tar.gz
+        run: |
+          mvn -B -ntp license:check install -DskipTests
```

Comment:
-Pcore-modules

Reply:
license check applies to all modules so we can't filter